docs/user_guide/1_setting_up_the_hand.md

```diff
@@ -187,18 +145,16 @@ You can check the currently available containers using:
 $ docker ps -a
 ```
 
-The container will be ready when fingers move to the zero position.
-
 ### Saving log files and uploading data to our server
 When running the one-liner, along with the icon that starts the Dexterous Hand, you will also notice a second icon named Save logs that is used to retrieve and copy all the available logs files from the active containers locally on your Desktop. This icon will create a folder that matches the active container's name and the next level will include the date and timestamp it was executed. When it starts, it will prompt you if you want to continue, as by pressing yes it will close all active containers. After pressing "yes", you will have to enter a description of the logging event and will start coping the bag files, logs and configuration files from the container and then exit. Otherwise, the window will close and no further action will happen. If you provided an upload key with the one-liner installation then the script will also upload your LOGS in compressed format to our server and notify the Shadow's software team about the upload. This will allow the team to fully investigate your issue and provide support where needed.
 
 ### Starting the driver
 
 * **Shadow Hand Driver**
-  Launch the driver for the Shadow Hand using the desktop icon 'Shadow_Hand_Launcher' if the one-liner was executed using the ```launch_hand=true``` argument or at a terminal (in the container), type:
+  Launch the driver for the Shadow Hand using the desktop icon 'Shadow_Hand_Launcher' if the one-liner was executed using the ```launch_hand=true``` argument or at a terminal (inside the container), type:
 
   ```bash
-  $ roslaunch sr_ethercat_hand_config sr_rhand.launch
+  $ ssh -t user@nuc-control roslaunch sr_ethercat_hand_config sr_rhand.launch
```

Comment:
no, I don't recommend doing that. If you want to launch the hand locally, plug in the hand ethernet adapter to the laptop and use the Advanced Launch Icon : Launch Local Shadow Hand.

Reply:
If you must use an ssh script, get the latest one from aurora